src/ws-dropdown/dropdown-menu.js

```diff
@@ -318,8 +340,8 @@ export class DropdownMenu extends Component {
           />,
           <li className="dropdown-item-separator" key="parent-separator" />
         ]}
-        {(this.state.value && this.state.value.length) ? [
-          this.state.value.map((item, index) =>
+        {(this.state.value && this.state.value.length && (this.context.multiple || this.props.filterable)) ? [
```

Comment:
This looks like complicated and hard to read. Maybe refactoring it into a function like "renderXyZ()" so it is easier to understand parts of the render function more easily ?